addons/godot-firebase/dynamiclinks/dynamiclinks.gd

```diff
@@ -0,0 +1,61 @@
+# ---------------------------------------------------- #
+#                 SCRIPT VERSION = 1.0                 #
+#                 ====================                 #
+# please, remember to increment the version to +0.1    #
+# if you are going to make changes that will committed #
+# ---------------------------------------------------- #
+
+class_name FirebaseDynamicLinks
+extends Node
+
+const _authorization_header : String = "Authorization: Bearer "
+
+var request : int = -1
+
+var _dynamic_link_request_url : String = "https://firebasedynamiclinks.googleapis.com/v1/shortLinks?key=%s"
+
+var config : Dictionary = {}
```

Comment:
Should have an underscore at the beginning, same with auth just below it. Probably the same with request on line 13.

Reply:
@WolfgangSenff we should go through all the scripts. It looks like not all of them are uniform. I did push a commit that fixes it for this script.

I need to look into the request section as I might be doing it a different way in the future.